tests/test_table.py

```diff
@@ -45,10 +45,10 @@ def test_wb_table_final_data(self):
         if experiment.wb_table is not None:
             test_column_list = ['epoch', 'index', 'positive/negative', 'sent1', 'sent2', 'shared_word', 'word_distance', 'hinge_loss', 'split']
             test_row_list = list([
-                [0, 0, 'positive', 'What are the best examples of the golden ratio in everyday life ?', 'Can you give some unknown examples of the golden ratio in our daily life that not everyone knows ?', 'life', 'train'],
```

Comment:
Why change this testing data?

Reply:
Because `('glue','qqp')` is indexed and sorted differently than `quora`.  So the example loaded with this configuration is different.